recipes-bsp/imx-atf/imx-atf_1.5.0.bb

```diff
@@ -0,0 +1,57 @@
+# Copyright 2017-2018 NXP
+
+DESCRIPTION = "i.MX ARM Trusted Firmware"
+SECTION = "BSP"
+LICENSE = "BSD-3-Clause"
+LIC_FILES_CHKSUM = "file://${COREBASE}/meta/files/common-licenses/BSD-3-Clause;md5=550794465ba0ec5312d6919e203a55f9"
+
+PV = "1.5.0+git${SRCPV}"
+
+SRCBRANCH = "imx_4.9.123_imx8mm_ga"
+SRC_URI = "git://source.codeaurora.org/external/imx/imx-atf.git;protocol=https;branch=${SRCBRANCH}"
+SRCREV = "af3554fc6ea8c17710ffd37a14a25834fad19271"
+
+S = "${WORKDIR}/git"
+
+inherit pkgconfig deploy
+
+BOOT_TOOLS = "imx-boot-tools"
+
+SOC_ATF ?= "imx8qm"
+SOC_ATF_mx8qm = "imx8qm"
+SOC_ATF_mx8qxp = "imx8qxp"
+SOC_ATF_mx8mq = "imx8mq"
+SOC_ATF_mx8mm = "imx8mm"
```

Comment:
Please use:

```
PLATFORM ?= "Invalid"
PLATFORM_mx8qm = "imx8qm"
...
```

`SOC_ATF` doesn't communicates nothing useful. Instead, it is misleading. Using the `PLATFORM` allows for immediate recognition what it is about and also it is easy to grasp when looking below when it gets used.